nexus/src/external_api/params.rs

```diff
@@ -229,6 +229,19 @@ pub struct DiskIdentifier {
     pub disk: Name,
 }
 
+/*
+ * IMAGES
+ */
+
+/// Create-time parameters for an
+/// [`Image`](omicron_common::api::external::Image)
+#[derive(Clone, Debug, Deserialize, Serialize, JsonSchema)]
+pub struct ImageCreate {
+    /// common identifying metadata
+    #[serde(flatten)]
+    pub identity: IdentityMetadataCreateParams,
+}
```

Comment:
We need more here to actually be able to create the image.

Do we want this to be an enum implying "Please provide one of a {file, snapshot UUID, URL}"? Do we want that to all be done through the same endpoint?

Reply:
Added an enum of "snapshot UUID or URL". We can continue to add extra params as necessary.